webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue

```diff
@@ -161,6 +170,13 @@ export default {
       this.drawer = true;
     },
     saveGroup(event) {
+      const regex =  /^[a-zA-Z0-9-_]+$/;
```

Comment:
It is better if you handle this at Rest service level as it is already done for validating if name or label are empty.

Reply:
done